### PR TITLE
Contributions scoreboard

### PR DESCRIPTION
**Pull request by toastythetoaster**
## Purpose
Implement the contributions scoreboard route using the provided mock api data.

### Images
Light | Dark
|:----:|:----:|
image | image
image | image
## Related
[see yif-13](https://linear.app/yifferxyz/issue/YIF-13/contributions-scoreboard-page)

### CODE REVIEW

File: app/routes/contribute/scoreboard/index.tsx [hunk 1]
```diff
@@ -1,17 +1,232 @@
...
+          onClick={() => setShowPointInfo(!showPointInfo)}
+          className="w-fit h-fit text-blue-weak-200 dark:text-blue-strong-300 font-semibold"
+        >
+          {showPointInfo ? 'Hide' : 'Show'} point info{' '}
+          {showPointInfo ? <MdArrowDropUp /> : <MdArrowDropDown />}
+        </a>
+      </p>
+      {showPointInfo && <PointInfo />}
+      <div className="flex flex-col justify-center items-center w-fit mx-auto mt-8">
+        <div className="flex mb-6">
```
Comment (ragnarob) [inline app/routes/contribute/scoreboard/index.tsx:116]: Let's make this into a component! Will add as a task for later

File: app/routes/contribute/scoreboard/index.tsx [hunk 2]
```diff
@@ -1,17 +1,232 @@
...
+            color="primary"
+            className={
+              'rounded-l-none' + disabledClass + (showAllTime ? enabledClass : '')
+            }
+            onClick={() => setShowAllTime(true)}
+          />
+        </div>
+        {!showAllTime && (
+          <div className="flex justify-between w-full mb-2 text-lg">
+            <Button
```
Comment (ragnarob) [inline app/routes/contribute/scoreboard/index.tsx:138]: We should make an IconButton component as well. Will also add a task in the board.

File: .prettierrc.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ module.exports = {
   trailingComma: 'es5',
   singleQuote: true,
   tabWidth: 2,
-  printWidth: 100,
+  printWidth: 90,
```
Comment (ragnarob) [inline .prettierrc.js:5]: Did we not talk about this? 😛

**Review by ragnarob — CHANGES_REQUESTED** [review]
A couple of changes requested, and a few things that don't need changing but that we should replace with new UI components later

**Review by ErikMelton — APPROVED** [review]
LGTM, just need to fix the git history 😄

**Comment by toastythetoaster** [comment]
> LGTM, just need to fix the git history 😄

this branch is going to end up killing me

**Pull request merged**